pkgs/development/python-modules/diff_cover/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, jinja2, jinja2_pluralize, pygments,
+  six, inflect, mock, nose, coverage, pycodestyle, flake8, pyflakes, git,
+  pylint, pydocstyle }:
+
+buildPythonPackage rec {
+  pname = "diff_cover";
+  version = "1.0.2";
+
+  LC_ALL = "en_US.UTF-8";
```

Comment:
I think this is only needed for the tests

Reply:
Yes it is. Without setting this variable tests will fail. Where should I put this variable definition?

Without this variable tests fail, if this variable will be visible only during tests execution,
runtime execution may fail too without LC_ALL.

Reply:
The env var can be set in the `preCheck` hook. This env var won't be set during runtime anyway.

Reply:
This package supply some executable files. If I wrap this executables with `makeWrapper` and set `LC_ALL` in wrapper?